### PR TITLE
configure controller with environment variables

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn @codiebeulaine ready for review

**Comment by smn** [comment]
@codiebeulaine can you please do a first review?

File: freebasics/models.py [hunk 1]
```diff
@@ -43,3 +45,15 @@ class FreeBasicsTemplateData(models.Model):
     footer_position = models.IntegerField(default=6)
     controller = models.OneToOneField(
         FreeBasicsController, on_delete=models.CASCADE)
+
+    def to_env_dict(self):
+        return {
+            'CUSTOM_CSS_BODY_BACKGROUND_COLOR': self.body_background_color,
+            'CUSTOM_CSS_BODY_FONT_COLOR': self.body_color,
```
Comment (codiebeulaine) [inline freebasics/models.py]: @miltontony this is now called block_background_color

File: freebasics/models.py [hunk 2]
```diff
@@ -43,3 +45,15 @@ class FreeBasicsTemplateData(models.Model):
     footer_position = models.IntegerField(default=6)
     controller = models.OneToOneField(
         FreeBasicsController, on_delete=models.CASCADE)
+
+    def to_env_dict(self):
+        return {
+            'CUSTOM_CSS_BODY_BACKGROUND_COLOR': self.body_background_color,
```
Comment (codiebeulaine) [inline freebasics/models.py]: this is now called base_background_color

File: freebasics/models.py [hunk 3]
```diff
@@ -43,3 +45,15 @@ class FreeBasicsTemplateData(models.Model):
...
+            'CUSTOM_CSS_BODY_FONT_COLOR': self.body_color,
+            'CUSTOM_CSS_BODY_FONT': self.body_font_family,
+            'CUSTOM_CSS_ACCENT_1': self.body_font_family,
```
Comment (codiebeulaine) [inline freebasics/models.py]: shouldn't this be self.accent1? and self.accent2?

**Comment by codiebeulaine** [comment]
@miltontony did a first pass at the review. Not really understanding the marathon stuff. But kinda get what's happening as a whole.

**Comment by miltontony** [comment]
@codiebeulaine ready for re-review

**Comment by codiebeulaine** [comment]
@miltontony :+1: just one q: what is the reason for changing the url to a charfield?

**Comment by miltontony** [comment]
@codiebeulaine so instead of storing the full `happy-place.molo.site`, we should just store `happy-place` and we construct `happy-place.whateverdomain.we.choose.com` so that if we do decide to change the domain in the future, we'll only need to change it in our config and no need to change the code.

**Comment by codiebeulaine** [comment]
@miltontony cool! :+1:

**Pull request merged**